pages/dkp/konvoy/2.2/image-builder/create-azure-image/index.md

```diff
@@ -70,7 +70,8 @@ Extract the bundle and `cd` into the extracted `konvoy-image-bundle-$VERSION_$OS
 -   Set the `AZURE_CLIENT_SECRET` environment variable:
```

Comment:
I think we should change this a bit since there are now variables being set in this section:

```suggestion
-   Set the `AZURE_CLIENT_SECRET` and `AZURE_SUBSCRIPTION_ID` environment variables:
```